components/eam/src/chemistry/modal_aero/aero_model.F90

```diff
@@ -202,6 +198,20 @@ subroutine aero_model_init( pbuf2d, species_class, iflagaa )
          history_verbose_out=history_verbose, &
          convproc_do_aer_out = convproc_do_aer, & 
          resus_fix_out       = resus_fix    ) 
+    ! The unified convective transport/removal for aerosols does not
+    ! do gases yet, and convproc_do_gas is just a place holder.  For that reason,
+    !    (1) All of the "if ( convproc_do_aer .or. convproc_do_gas ) then" statements
+    !        in aero_model.F90 have been changed to "if ( convproc_do_aer ) then"
+    !    (2) convproc_do_aer=.false. and convproc_do_gas=.true. is no longer allowed.
+    !
+    ! for C++ porting: namelist [history_aerosol, history_verbose, convproc_do_aer, resus_fix]
+    ! are [T, F, T, T] in default. history_aerosol is kept as there is no code
+    ! related to history_aerosol=.false. All other conditions are removed
+    ! as they are not used/tested.
+    if (history_verbose) call endrun('history_verbose=.true. is removed')
+    if (.not. convproc_do_aer) call endrun('convproc_do_aer=.false. is removed')
+    if (.not. resus_fix) call endrun('resus_fix=.false. is removed')
+
```

Comment:
We might want to keep the history namelist variables. I was going to add a test to turn these on just to make sure we are not messing up any diagnostics.

Reply:
did you mean to keep only `history_verbose` or all the three namelist variables?

Reply:
Just keep the `history_verbose`.